src/scenario_tests.rs

```diff
@@ -968,7 +968,35 @@ mod dst_tests {
                 ParsecVote::CheckElder.to_event(),
             ],
             &AssertState {
-                action_our_events: vec![Rpc::Split.to_event()],
+                action_our_events: vec![
+                    ParsecVote::NewSectionInfo(SectionInfo(Section(1), 0)).to_event(),
```

Comment:
maybe good to have a constant for `SectionInfo(Section(1), 0)` and the other to use in both tests.